src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.cs

```diff
@@ -827,7 +829,7 @@ private void UpdateInlines(string text)
 				return;
 			}
 
-			hyperlink.SetPointerPressed(e.Pointer);
+			hyperlink.SetPointerPressed(e.Pointer, that.Resources.ThemeDictionaries);
```

Comment:
No need to pass this I think. You can probably use `Application.Current.Resources`